libs/langchain-community/langchain.config.js

```diff
@@ -288,6 +290,8 @@ export const config = {
     "utils/convex": "utils/convex",
     "utils/event_source_parse": "utils/event_source_parse",
     "utils/cassandra": "utils/cassandra",
+    "utils/tencent_hunyuan/web": "utils/tencent_hunyuan/web",
+    "utils/tencent_hunyuan/index": "utils/tencent_hunyuan/index",
```

Comment:
Remove the first `/index`:

`"utils/tencent_hunyuan": "utils/tencent_hunyuan/index",`